docs/run/advanced-runtime-options.mdx

```diff
@@ -129,7 +129,7 @@ sampler.run([circuit1, circuit2], shots=128)
   </TabItem>
 
   <TabItem value="PrimV1" label="V1 primitives">
-You can pass any options to `run()`. Because most users will only overwrite a few options at the job level, it is not necessary to specify the category the options are in. The code below, for example, specifies `shots=1024` instead of `execution={"shots": 1024}` (which is also valid). 
+You can pass any options to `run()`. Because most users will only overwrite a few options at the job level, it is not necessary to specify the category the options are in. The code below, for example, specifies `shots=1024` instead of `execution={"shots": 1024}` (which is also valid).
```

Comment:
```suggestion
You can pass any options to `run()`. Because most users will only overwrite a few options at the job level, it is not necessary to specify the option category. The code below, for example, specifies `shots=1024` instead of `execution={"shots": 1024}` (which is also valid).
```